src/common/set.c

```diff
@@ -2213,7 +2272,7 @@ util_header_create(struct pool_set *set, unsigned repidx, unsigned partidx,
 		1, POOL_HDR_CSUM_END_OFF);
 
 	/* store pool's header */
-	util_persist_auto(rep->part[partidx].is_dev_dax, hdrp, sizeof(*hdrp));
```

Comment:
`util_persist(is_pmem, ...);`

the "auto" function when far away from my original intention...